podcasts/PlaylistsViewController+Table.swift

```diff
@@ -24,14 +24,25 @@ extension PlaylistsViewController: UITableViewDelegate, UITableViewDataSource {
     }
 
     func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
-        return FeatureFlag.playlistsRebranding.enabled ? PlaylistCell.cellHeight : FilterNameCell.cellHeight
+        return FeatureFlag.playlistsRebranding.enabled ? NewPlaylistCell.cellHeight : FilterNameCell.cellHeight
     }
 
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         if FeatureFlag.playlistsRebranding.enabled {
-            let cell = cell(tableView, for: PlaylistCell.reuseIdentifier) as! PlaylistCell
+            let cell = cell(tableView, for: NewPlaylistCell.reuseIdentifier) as! NewPlaylistCell
             if let playlist = playlists[safe: indexPath.row] {
-                cell.configure(playlist: playlist, isLastRow: indexPath.row == playlists.count - 1)
+                if cell.tag != indexPath.row { cell.resetAllMetadata() }
+                cell.tag = indexPath.row
+                cell.set(playlistName: playlist.playlistName, isManualPlaylist: playlist.manual)
+                Task { [weak self] in
+                    await self?.playlistMetadataLoader.loadMetadata(for: playlist) { count in
+                        guard let cell = tableView.cellForRow(at: indexPath) as? NewPlaylistCell else { return }
```

Comment:
@bjtitus @SergioEstevao any better idea to avoid detecting the cell for that specific indexpath when the block is called and set the value? This avoids setting values in wrong cells